docs/useGetIdentity.md

```diff
@@ -23,7 +23,7 @@ Once loaded, the `data` object contains the following properties:
 const { id, fullName, avatar } = data;
 ```
 
-`useGetIdentity` uses [react-query's `useQuery` hook](https://tanstack.com/query/v3/docs/react/reference/useQuery) to call the `authProvider`.
+`useGetIdentity` uses [react-query's `useQuery` hook](https://tanstack.com/query/v3/docs/react/reference/useQuery) to call the `authProvider`. This means you will need to wrap the component where you want to use it in a react-query `<QueryClientProvider>` with a default `QueryClient`. To use `useGetIdentity()` in your `App.tsx` you wrap it in `index.tsx`.
```

Comment:
Thanks for your contribution.

I get your point, and I agree it's a good idea to document this gotcha.

However I disagree with the place and way you chose to do it:
- Using `useGetIdentity` outside the `<QueryClientProvider>` is rather uncommon. Most of the time, calling this hook will work with no extra step => This additional piece of information should rather be added as a **Tip**. Besides, it should be moved to the **Usage** section and not the **Syntax** section.
- ``To use `useGetIdentity()` in your `App.tsx` you wrap it in `index.tsx`.`` This sentence makes little sense to me, and relies on conventions regarding the naming of the files that are not standard. I'd recommend removing it. Mentioning that `useGetIdentity` needs to be used inside a `<QueryClientProvider>` is probably enough.

=> Can you please move the first part of your tip to the **Usage** section, and make it a **Tip**?